xscen/ensembles.py

```diff
@@ -49,11 +57,17 @@ def ensemble_stats(
     xr.Dataset
         Dataset with ensemble statistics
     """
-    if stats_kwargs is None:
-        stats_kwargs = {}
-    if create_kwargs is None:
-        create_kwargs = {}
-    logger.info(f"Creating ensemble with {len(datasets)} and calculating {statistics}.")
+    create_kwargs = create_kwargs or {}
+    logger.info(
+        f"Creating ensemble with {len(datasets)} simulations and calculating {statistics}."
+    )
+
+    if isinstance(statistics, str) and isinstance(stats_kwargs, dict):
+        logger.warning(
+            "DeprecationWarning: The usage of 'statistics: str' with 'stats_kwargs: dict' will be abandoned. Please use 'statistics: dict' instead."
+        )
```

Comment:
```suggestion
        warnings.warn(
            "The usage of 'statistics: str' with 'stats_kwargs: dict' will be abandoned. Please use 'statistics: dict' instead.",
            category=FutureWarning
        )
```
Voir ceci pour la différence entre Deprecation et Future :https://docs.python.org/3/library/warnings.html#warning-categories
Dans le fond, Deprecation c'est pour les dev, Future pour les utilisateurs.